test-tools/host-tool/src/transport.h

```diff
@@ -12,8 +12,8 @@ extern "C" {
 
 /* IMRT link message between host and WAMR */
 typedef struct {
-    unsigned short message_type;
-    unsigned long payload_size;
+    uint16_t message_type;
```

Comment:
uint16_t requires <stdint.h>, normally we don't use uint16_t, uint32_t type except in core/iwasm/include/wasm_export.h, could you please remain it as unsigned short?